packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -787,7 +812,7 @@ function makeAugmentedSchema(
 
         composer.createInputTC({
             name: `${node.name}Where`,
-            fields: queryFields,
+            fields: node.isGlobalNode() ? { id: "ID", ...queryFields } : queryFields,
```

Comment:
If the user opts into using the global node, an ID field will be generated for it and returned. Without adding the "id" field to where, the only way the user can query a specific node by identity would be to call the top-level `node(id: ID!): Node!` query. If we add the id field to the where argument, users can query a Movie by "id"

Hypothetically, the user could manage the retrieval and querying of unique ids by themselves -- e.g., returning the underlying unique field and then querying for that. But one of the benefits of relay is having and using a single unique id, rather than having to request two different identifiers and decide which is appropriate "where" (pun intended).

Reply:
Sure, I'm convinced!!